frontend/plan/components/FilterButton.tsx

```diff
@@ -0,0 +1,83 @@
+/* A button that toggles on click and allows search to filter class that only fits schedule */
```

Comment:
nit: lets move this file to under the `search` folder and rename to `ButtonFilter` to be consistent with the other filter components.